apps/full-site-editing/full-site-editing-plugin/wp-template.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * Mostly a copy of how the `wp_block` is registered in core
+ *
+ * @return void
+ */
+function fse_register_wp_template() {
+	register_post_type(
+		'wp_template',
+		array(
+			'labels'                => array(
+				'name'                     => _x( 'Templates', 'post type general name' ),
+				'singular_name'            => _x( 'Template', 'post type singular name' ),
+				'menu_name'                => _x( 'Templates', 'admin menu' ),
+				'name_admin_bar'           => _x( 'Template', 'add new on admin bar' ),
+				'add_new'                  => _x( 'Add New', 'Template' ),
+				'add_new_item'             => __( 'Add New Template' ),
+				'new_item'                 => __( 'New Template' ),
+				'edit_item'                => __( 'Edit Template' ),
+				'view_item'                => __( 'View Template' ),
+				'all_items'                => __( 'All Templates' ),
+				'search_items'             => __( 'Search Templates' ),
+				'not_found'                => __( 'No Templates found.' ),
+				'not_found_in_trash'       => __( 'No Templates found in Trash.' ),
+				'filter_items_list'        => __( 'Filter Templates list' ),
```

Comment:
Dang, must have forgot to submit this other nitpick earlier.
Not found and filter labels should have `template` lowercase: https://github.com/WordPress/WordPress/blob/8e9315f51e4dd60ee77ef7b2d4df0fe8327a9e3b/wp-includes/post.php#L270-L272

Reply:
Nice catch again. No more blind search/replace for me next time. :)